.github/workflows/_temp_usage_counter_family_repair.yml

```
@@ -0,0 +1,112 @@
+      - name: Support Chat and Responses token families
+        shell: bash
+        run: |
+          set -euo pipefail
+          python - <<'PY'
+          from pathlib import Path
+
+          path = Path("contextual_orchestrator/orchestrator.py")
+          text = path.read_text(encoding="utf-8")
+
+          old_positive = '''    @staticmethod
+    def _usage_has_positive_evidence(usage: Any) -> bool:
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)